### PR TITLE
Adding a contributing guide

### PR DESCRIPTION
**Pull request by JanWeldert**
First draft of a contributing guide. Still needs some fine tuning, but that would be my idea of the overall structure.

_Not sure why the PR is showing some old commits, only the last one is important here_

### CODE REVIEW

**Comment by JanWeldert** [comment]
Closes #825

File: docs/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# How to contribute to PISA
+
+Thank you for reading this and for considering to contribute to PISA! We always need volunteer developers to help move this project forward.
+
+## Submitting changes
+Found a bug or missing a functionality? Don't be afraid to tackle this! If you don't do it, maybe no one will.
```
Comment (thehrh) [inline docs/CONTRIBUTING.md:6]: "missing a feature" sounds more idiomatic to me here

File: docs/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+#### Create an issue
+In any case, it is a good first step to document what you've found (or what's missing) in a GitHub issue. [Here](https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-an-issue) is how-to-do it.
```
Comment (thehrh) [inline docs/CONTRIBUTING.md:9]: * remove contractions here for consistency: you have, what is
* how to do it

File: docs/CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+#### Close an issue
+If you want to work on something already documented, you are more than welcome! But remember, the best way to close an issue is with a pull request.
```
Comment (thehrh) [inline docs/CONTRIBUTING.md:12]: * straight to the point: "You are more than welcome to ..."

File: docs/CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+#### Open a pull request
+Once you implemented your changes, you can start a PR. You can start this early, even when you are still working on your code, to get feedback and let people know what you are working on. Use your own fork and/or a dedicated branch for your PR. [Here](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request) is how-to-do it.
```
Comment (thehrh) [inline docs/CONTRIBUTING.md:15]: * I think it's fine to suggest to use own fork _and_ dedicated branch
* how to do it

File: docs/CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+#### Review and merge a pull request
+Everyone can (and should) review and merge PRs opened by other contributors. You can also just review and wait for others to merge, if you are unsure about the proposed changes.
```
Comment (thehrh) [inline docs/CONTRIBUTING.md:18]: We have no instructions on how to best merge a PR currently. For example, this PR has a number of unnecessary commits, which we should be able to get rid of with a squash merge. Maybe refrain from explicitly suggesting that everyone should perform merges for now (likely that only org members are able to do so anyway).

File: docs/CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+Everyone can (and should) review and merge PRs opened by other contributors. You can also just review and wait for others to merge, if you are unsure about the proposed changes. 
+
+## Coding conventions
+Coding conventions can be found in [Conventions](pisa/general_conventions.md).
+
+## Testing
+If you add a feature, add a test that proves it works. More about tests can be found in [Conventions](pisa/general_conventions.md).
+
+## Ask for help
+If you are stuck with you code or have some generel questions, feel free to ask other people on slack (e.g. in the pisa-help channel).
```
Comment (thehrh) [inline docs/CONTRIBUTING.md:27]: * you -> your, generel -> general
* some comment that acknowledges that this repo is public while the slack channel you're referring to isn't?

**Review by thehrh — CHANGES_REQUESTED** [review]
see inline comments

**Review by thehrh — APPROVED** [review]

**Pull request merged**